src/test/scala/org/broadinstitute/dsde/firecloud/service/EntityServiceSpec.scala

```diff
@@ -166,6 +178,22 @@ class EntityServiceSpec extends ServiceSpec with EntityService {
       }
     }
 
+    "when calling GET on valid entityQuery with no params" - {
+      "OK response is returned" in {
+        Get(validFireCloudEntityQuerySamplePath) ~> dummyAuthHeaders ~> sealRoute(routes) ~> check {
+          status should be(OK)
+        }
+      }
+    }
+
+    "when calling GET on valid entityQuery with params" - {
+      "OK response is returned" in {
+        Get(validFireCloudEntityQuerySamplePath + "?page=1&pageSize=1") ~> dummyAuthHeaders ~> sealRoute(routes) ~> check {
+          status should be(OK)
+        }
+      }
+    }
+
```

Comment:
There's not much to these tests, and there shouldn't be, since orchestration's API is effectively deferring to rawls implementation. These tests really just check that orch's routes are correct.